scripts/code_style.py

```diff
@@ -137,6 +139,14 @@ def get_src_files(since: Optional[str]) -> List[str]:
                          is_file_autogenerated(filename))]
     return src_files
 
+def get_submodule_hash(commit: str, submodule: str) -> str:
+    """
```

Comment:
Not a blocker because we don't use a documentation processing tool, but [PEP 8](https://peps.python.org/pep-0008/#documentation-strings) says to write the first line of the docstring immediately after the opening `"""`.

Reply:
For future reference (it doesn't matter for this time), in this project, the author of a review comment is the one who's supposed to mark it as resolved, when they're happy about the resolution. As a pull request author, if you want to keep track of which comments you've handled, we use the :rocket: reaction (you don't need to do it if you don't find it useful).